docs/www/kubernetes-external-connect.md

```diff
@@ -107,6 +111,26 @@ We'll use Helm to install cert-manager:
     redpanda/redpanda-operator
   ```
 
+  </tab>
+
+	<tab id="zsh">
+
+
+  ```
+  noglob kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=$VERSION && \
```

Comment:
I have no strong opinion one way or another, but the onus of using `noglob` seems like it should on those using such shells.

Reply:
I don't disagree but it has been something that has tripped up a user or two and is very confusing for this command specifically as it is using an ability of kustomize to pull down a github link and not a normal HTTP link.